app/javascript/blacklight/index.js

```diff
@@ -11,7 +10,6 @@ export default {
   Autocomplete,
   BookmarkToggle,
   ButtonFocus,
-  CheckboxSubmit,
```

Comment:
There still is a checkbox_submit.js with an export, why do these two lines get removed from index.js, is that right?

Reply:
Because it is imported by bookmark_toggle.js, which is the only place it is used.

Reply:
Makes sense! Is this possibly a backwards compat change, if a local app was using it?  If so, that's maybe okay since we're shooting for major version?  Just wanted to be clear (and is there anywhere we're listing backwards compat changes?)

Reply:
That was never available in a release. That file was just created 2 weeks back: https://github.com/projectblacklight/blacklight/commit/4976e2d34539a5eb90a7de40689357fc9711e66e